### ISSUE TITLE
cachePath doesn't reflect correct router urls

### ISSUE THREAD

**Issue by jonaslund** [pajtai/express-static-file-cache#2]
``` js
var cachePath           = path.join(cacheDir, req.url);
```

Doesn't reflect the correct paths when using express 4.0 router functions

``` js
var router = require("./routes/router");
app.use("/router/", router);

```

```
router.get("/", function(req, res { 
});
```

should be

``` js
cachePath = "/router/index.html";
```

 but only gets index.html

**Comment by jonaslund** [comment]
One solution is to use

``` js
var cachePath           = path.join(cacheDir, req.originalUrl);
```

or 

``` js
var cachePath           = path.join(cacheDir, req.baseUrl + req.url);
```

depending on your router needs

**Comment by pajtai** [comment]
@jonaslund good catch on this one. I'm a little slammed at work now... just wanted to say I appreciate the feedback and I'll try and get a bug fix in soon.

**Comment by pajtai** [comment]
closed by #4
